### PR TITLE
Add document iteration puzzle

### PR DESCRIPTION
**Pull request by devmake**

### CODE REVIEW

File: www/puzzles/documents/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<div id="message">
+    Welcome dear user of our amazing document management system. We do not recommend storing any passwords or secrets in your documents as permission checks are still not implemented yet. We were notified that some users store their passwords despite our warning...
+</div>
+<hr>
+
+<div id="main">
+    <h2>My Documents</h2>
+    <div>
+        <a href="/getDocument?id=1" target="_blank">Work.txt</a>
+        <a href="/getDocument?id=4" target="_blank">Notes.txt</a>
```
Comment (kacz) [inline www/puzzles/documents/index.html]: Was it a deliberate choice that Notes.txt opens an empty file?

File: www/puzzles/documents/index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <a href="/getDocument?id=7" target="_blank">Programming.txt</a>
```
Comment (kacz) [inline www/puzzles/documents/index.html]: Would be nice to show each file on a separate line:
```suggestion
        <a href="/getDocument?id=1" target="_blank">Work.txt</a><br>
        <a href="/getDocument?id=4" target="_blank">Notes.txt</a><br>
        <a href="/getDocument?id=7" target="_blank">Programming.txt</a><br>
```

File: www/puzzles/documents/index.html (hunk 1 above)
Comment (devmake) [reply www/puzzles/documents/index.html]: Well, it was not. I added the content now. Thanks

File: www/puzzles/documents/index.html (hunk 2 above)
Comment (devmake) [reply www/puzzles/documents/index.html]: I personally liked the way they looked but I don't mind too much. I added the break lines.

**Review by kacz — APPROVED** [review]

**Comment by kacz** [comment]
Really nice puzzle, thanks!

**Pull request merged**